python/gui/auto_generated/layout/qgslayoutdesignerinterface.sip.in

```diff
@@ -109,6 +109,13 @@ Returns whether the atlas preview mode is enabled in the designer.
 .. seealso:: :py:func:`setAtlasPreviewEnabled`
 
 .. versionadded:: 3.4
+%End
+
+    virtual void setAtlasFeature( const QgsFeature &feature ) = 0;
+%Docstring
+Sets the specified feature as the current atlas feature
```

Comment:
\warning it is the caller's responsibility to ensure that \a feature is a feature from the layout's current atlas coverage layer.